src/tools/genpolicy/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "genpolicy"
+version = "0.1.0"
+authors = ["The Confidential Containers community https://github.com/confidential-containers"]
+edition = "2021"
+
+[dependencies]
+# Logging.
+env_logger = "0.10.0"
+log = "0.4.17"
+
+# Command line parsing.
+clap = { version = "4.1.8", features = ["derive"] }
+
+# YAML file serialization/deserialization.
+base64 = "0.21.0"
+serde = { version = "1.0.159", features = ["derive"] }
+
+# Newer serde_yaml versions are using unsafe-libyaml instead of yaml-rust,
+# and incorrectly change on serialization:
+#
+# value: "yes"
+#
+# to:
+#
+# value: yes
+#
+# In YAML, the value yes without quotes is reserved for boolean,
+# and confuses kubectl, that expects a string value.
+serde_yaml = "0.8"
+
+# Container repository.
+anyhow = "1.0.32"
+async-trait = "0.1.68"
+docker_credential = "1.2.0"
+flate2 = { version = "1.0.26", features = ["zlib-ng"], default-features = false }
+oci-distribution = { version = "0.9.4" }
```

Comment:
> 1. One of my colleagues at MSFT noticed the ubuntu:latest problem a few days ago, and we are working on a solution for it. I can share the gory details in case you are curious about the root cause. Meanwhile, creating your own container image on top of an Ubuntu base container image (but using a different manifest format) might be a good enough workaround.

Hey @danmihai1, I noticed the problem with ubuntu:latest, which was caused by the manifest type. We should handle manifests with the mediatype “application/vnd.oci.image.manifest.v1+json” when pulling them from the registry. This issue had been solved in the [oci-distribution](https://github.com/krustlet/oci-distribution/pull/67). Could you please change the oci-distribution repo branch to [this commit](https://github.com/krustlet/oci-distribution/pull/67/commits/0036d93e8f6eb1df4e01e60c4f1c9c77ea43461c)?”

Reply:
@ChengyuZhu6 @katexochen Christmas came early for me this year and brought a new release of the oci_distribution crate - a few days ago. Therefore, I have just added to this PR @Redent0r's commit that adds support for the ubuntu:latest image by using the new oci_distribution version.